electron_app/src/lang/es.json

```diff
@@ -93,115 +81,104 @@
       "description": {
         "prefix": "El tamaño del archivo",
         "suffix": "es mayor a",
-        "defaultEnd": "l tamaño permitido"
```

Comment:
without `l` it will say `El tamaño del archivo es mayor a tamaño permitido`

Reply:
It would be nice if we can come up with a function like `console.log("Hi %s", "Erika")` so we can have full semantic strings here